### PR TITLE
[IMP] live chat:update getting started doc

### PR DESCRIPTION
**Pull request by jero-odoo**
This PR updates the Live Chat "Getting Started with Live Chat" document. This includes expanding on the setup details and adding context to each of the steps involved. Also adds information for users participating in live chat. Updates screenshots and terms to match 16.0.

### CODE REVIEW

File: content/applications/websites/livechat/overview/get_started.rst
Comment (ksc-odoo) [inline content/applications/websites/livechat/overview/get_started.rst:170]: To join a :guilabel:`Live Chat` channel, go to the :menuselection:`Live Chat` app and click the
:guilabel:`JOIN` button on the kanban card for the appropriate channel. 

Any channel where the user is currently active will show a :guilabel:`LEAVE` button. Click this button to disconnect from the channel.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (ksc-odoo) [inline content/applications/websites/livechat/overview/get_started.rst:177]: *Operators*?

Comment (ksc-odoo) [reply content/applications/websites/livechat/overview/get_started.rst:177]: ...will be considered "disconnected," and subsequently removed from the channel.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (ksc-odoo) [inline content/applications/websites/livechat/overview/get_started.rst:184]: I think it would be beneficial to the overall doc if you took the time to expand on this section. Talk about how to navigate to the Discuss app - briefly touch on the Discuss interface/dashboard - where to look for specific Live Chat communications - etc.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (ksc-odoo) [inline content/applications/websites/livechat/overview/get_started.rst:192]: Would this section be better suited up where the *operators* are first mentioned (and explained)? Seems a bit disjointed at this part of the doc. What do you think?

**Review by ksc-odoo — COMMENTED** [review]
Hey @jero-odoo - just finished my Peer Review of this doc. Great work! In general, I think it would benefit this overall doc if certain elements/aspects were more "flushed out"/expanded upon. Leave no stone unturned, so to speak. If a button creates a pop-up window, mention that, and also mention every single field in that pop-up window, what it does, what the options are, etc. --- just expand on explanations wherever possible to avoid any confusion. If you have any questions, or need clarification about anything, don't hesitate to reach out. Thanks!

File: content/applications/websites/livechat/overview/get_started.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/overview/get_started.rst:44]: Because in this case I am talking about the role "operators" in the abstract, and not something specific to the UI, I am not going to add the guilabel.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/overview/get_started.rst:66]: The times where the guilabel is used is times where it is directly related to a item in the UI. The times where it is not is when I am referring to the concept of "live chat" in general. I realize there are some instances where it may seem a bit arbitrary, but I do not want to over load the document with guilabels when they are not necessary.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/overview/get_started.rst:68]: I agree. I wanted to keep the color selections at the end (because they required additional information) but it does make sense to move the button information up for the doc flow.

File: content/applications/websites/livechat/overview/get_started.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/overview/get_started.rst:142]: Again, I realize it can seem arbitrary, however I try to keep it separate between actual UI items (which require the guilabel) and concepts (which do not). I will make sure to take another look to ensure that there is consistency between these two, however, I have been told not to add a guilabel unless specifically referring to a UI item.

**Comment by jero-odoo** [comment]
Hi @samueljlieber, can you take a look at this before it goes over to @StraubCreative? Thank you!
Quick note-Live Chat is the app, I used guilabel/italics when referring to that. However, when referring to the concept/act of participating in a 'live chat', I kept it lower case. This is both keeping with industry trends, and matches the way we use the term currently (both for our internal live chat process, and on the website).

File: content/applications/websites/livechat/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/websites/livechat/overview/get_started.rst:280]: all images - we dont need to prepend `live-chat-` to each image filename since it is already implied by residing within the folder

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @jero-odoo 👋 I've made some technical changes on this PR as well as fixed the merge conflicts. I will implement these changes in a commit after this review, please review the changes and let me know if you approve and I will pass this to final review!

Summary of my changes:
- Italicization when not an active instruction or introducing a concept for the first time & :guilabel: when it is an active instruction
- Using :menuselection: when directing the user to an app
- Bullet point indentation: make sure to stay aligned/no need to indent one space
- Image filenames: no need to prepend `live-chat-`

With fixing the merge conflicts I had to remove the `overview` folder and corresponding rst file, remove the majority of the text from livechat.rst except for the :toc: content, remove any images that were in overview or livechat folders, and re-add redirects for those changes.

Great job with this doc, and let me know if you approve of these changes! Thank you 🙂

**Comment by samueljlieber** [comment]
Implemented technical changes in https://github.com/odoo/documentation/commit/34d3f1817b8454ee51358fe1fcff63c48ac666bc

File: content/applications/websites/livechat/overview/get_started.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/overview/get_started.rst]: It was off in the build as well, not just in the VS file. Can you confirm it is working without the escape?

Comment (samueljlieber) [reply content/applications/websites/livechat/overview/get_started.rst]: Yes can confirm it is working without the escape :)

**Comment by samueljlieber** [comment]
@jero-odoo thank you for the approval - and yes I confirmed the use of a `:` works within a :guilabel: without escaping, thank you for double checking 🙂

Hi @StraubCreative 👋 this PR is ready for your final review!

**Review by samueljlieber — APPROVED** [review]

**Review by StraubCreative — CHANGES_REQUESTED** [review]
@jero-odoo please fix 100th character line break issues and tag me again, thanks!

**Comment by jero-odoo** [comment]
@StraubCreative Updated! Thank you 👍

**Comment by xpl-odoo** [comment]
Hello @jero-odoo!

As part of the [User Docs Structure Review](https://github.com/orgs/odoo/teams/user-docs-structure-review), I'm notified whenever a new "app-level" page is added to check if the structure is alright. 

Here I would move the content from `get_started.rst` to `livechat.rst`, as the main app page is supposed to introduce major features and workflows.

Thanks!

**Comment by StraubCreative** [comment]
> Hello @jero-odoo!
> 
> As part of the [User Docs Structure Review](https://github.com/orgs/odoo/teams/user-docs-structure-review), I'm notified whenever a new "app-level" page is added to check if the structure is alright.
> 
> Here I would move the content from `get_started.rst` to `livechat.rst`, as the main app page is supposed to introduce major features and workflows.
> 
> Thanks!

Agree with @xpl-odoo here. There's a larger effort repo-wide to reduce the number of clicks users have to take to consume content, especially for intro/get started content. That information should be reserved for top-level app pages, instead of wiping out top-level page and creating a new doc.

**Comment by jero-odoo** [comment]
Thanks @StraubCreative, I made the requested changes. Please hold off on final review until @samueljlieber has a chance to do another tech review just to confirm everything is correct 👍
Sam-let me know if everything looks good, thanks!

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:31]: I think we can lowercase live chat here
```suggestion
Create a new live chat channel
```

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:67]: No need for a guilabel here
```suggestion
New operators can be created and added to the list directly from this pop-up, as well,
```

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:97]: Need a guilabel here
```suggestion
window. Click the :guilabel:`refresh` icon to the right of the color bubbles to reset the colors to the default
```

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:118]: I think the capitalization on _channel header_ is incorrect, Color is capitalized in the UI for **Channel Header Color**, and _Livechat Button Color_ should be italicized

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:129]: Please add guilabels to the items with active instruction here rather than italics

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:161]: I feel the guilabel on chatbot here is unecessary since we are talking about a chatbot in general

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:186]: I dont think we should add a guilable to the database types, italics is probably best

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:192]: We should guilabel **Widget** tab here since it is part of the active instruction breaking down the Live Chat Channel form

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:196]: Shown as one word in the UI
```suggestion
:menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Livechat`
```

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:224]: I dont believe profile page needs a guilabel

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:270]: menu bar should be lowercase, and Im indecisive if it needs a guilabel - curious on @StraubCreative 's thoughts here

Comment (jero-odoo) [reply content/applications/websites/livechat.rst:270]: I changed it to lowercase and removed the guilabel-because while yes it is part of the UI, it is location, not the actual part that the user is interacting with. @StraubCreative let me know your thoughts

**Comment by jero-odoo** [comment]
@samueljlieber I made the changes you requested, can you review before passing this to final? Thank you!

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:19]: I believe this is writing style, so feel free to ignore this comment but I don't feel it is necessary to include **Main Odoo Dashboard** every time in the navigation between apps/modules, @StraubCreative wdyt?

File: content/applications/websites/livechat.rst
Comment (samueljlieber) [inline content/applications/websites/livechat.rst:276]: If we're including the dashboard in the menuselection navigation I feel we should keep the naming consistent throughout the doc
```suggestion
Live chat conversations can also be viewed by navigating to :menuselection:`Main Odoo Dashboard --> Discuss`.
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo great job with this PR! Im approving for final review with two outstanding notes on menuselection formatting. 🙂 @StraubCreative

**Comment by jero-odoo** [comment]
@StraubCreative This is ready for your review. Thank you!

File: content/applications/websites/livechat.rst
Comment (StraubCreative) [inline content/applications/websites/livechat.rst:111]: Suggestion
```suggestion
Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open a new chat
session. This message will appear as though it is sent by a live chat operator, and acts as both a
greeting and an invitation to continue the conversation.
```

**Review by StraubCreative — APPROVED** [review]
Hi @jero-odoo 
Nice doc 🔥 
I'm approving with a few suggestions, which you can choose to accept or ignore at your discretion.
When you're ready go ahead and run the merge command, thanks!

@robodoo delegate=jero-odoo

File: content/applications/websites/livechat.rst
Comment (jcs-odoo) [reply content/applications/websites/livechat.rst:19]: The guidelines aren't very clear and could use a long overdue update, but it's there as an example in the table. ^^'
```
:menuselection:`App name --> Menu --> Sub-menu`
```
https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#document-structure

Similarly, you would group the menu selection in the next bullet point:
```
Go to :menuselection:`Website --> Configuration --> Settings`, check the box next to :guilabel:`Livechat`,
  and click :guilabel:`Save`.
```

Also, in general, we try not to reexplain everything everywhere: it's best to use custom links or "see also" sections, such as.
```
:ref:`Install <general/install>`
```

File: content/applications/websites/livechat.rst
Comment (jero-odoo) [reply content/applications/websites/livechat.rst:73]: My editor does show it differently than other guilabels, but it doesn't seem to show an issue in the HTML. I tried to break it out with the escape markup, but that seemed to just make things messier.

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
